sycl/test-e2e/Matrix/joint_matrix_colA_rowB_colC.cpp

```diff
@@ -5,13 +5,13 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+// VNNI transform and sub-group size 32 are not supported yet on DG2 by IGC
+// UNSUPPORTED: gpu-intel-dg2
 // REQUIRES: aspect-ext_intel_matrix
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// XFAIL:*
```

Comment:
Does this test pass on PVC? I did not see any unexpected pass on PVC for this test...